libs/libvtrutil/src/vtr_vector.h

```diff
@@ -61,7 +61,6 @@ class vector : private std::vector<V, Allocator> {
     using storage::size;
 
     using storage::back;
-    using storage::data;
```

Comment:
Why is the data() member removed? It is part of C++11.

Reply:
I removed it to fix this error:
```
In file included from /Users/deweese/src/vtr-verilog-to-routing/libs/libvtrutil/src/vtr_string_interning.h:54:
/Users/deweese/src/vtr-verilog-to-routing/libs/libvtrutil/src/vtr_vector.h:64:20: error: no member named 'data' in 'std::__1::vector<bool, std::__1::allocator<bool> >'
    using storage::data;
          ~~~~~~~~~^
/Users/deweese/src/vtr-verilog-to-routing/vpr/src/draw/draw_types.h:205:39: note: in instantiation of template class 'vtr::vector<vtr::StrongId<cluster_block_id_tag, int, -1>, bool,
      std::__1::allocator<bool> >' requested here
    vtr::vector<ClusterBlockId, bool> use_default_block_color_;
                                      ^
```
Maybe the problem is in `draw_types.h` instead, since it's that same error repeated. I'll look at it some more.

Reply:
Hmm, `std::vector<bool>` is [has a proxied implementation in the C++ standard](https://stackoverflow.com/questions/17794569/why-is-vectorbool-not-a-stl-container) which means it isn't really a container. Comparing the definitions of [std::vector\<T\>](http://www.cplusplus.com/reference/vector/vector/) and [std::vector\<bool\>](http://www.cplusplus.com/reference/vector/vector-bool/) show significant differences in the availability of different member functions.

Probably the fix is to only enable the:
```cpp
using storage::data;
```
in vtr::vector only if `V` is not a `bool` (perhaps with `std::enable_if`?).

Reply:
This is now fixed by #1282. Thanks @hzeller!